content/en/_index.html

```diff
@@ -43,6 +43,41 @@ <h3 class="section-head">
   </div>
 </div>
 
+<div id="adopters" class="text-center">
+  <div class="container">
+    <h3 class="section-head">
+      <span class="border-bottom pb-2">Featured Adopters of Kubeflow</span>
```

Comment:
Can we rename this to `Trusted By` ?
This is well established in CNCF projects:
- https://istio.io/
- https://argoproj.github.io/
- https://knative.dev/docs/
```suggestion
      <span class="border-bottom pb-2">Trusted By</span>
```